requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy>=1.14
+scipy>=0.18.1
+mne>=0.19.1
+nibabel>=2.2
+pybv
+pytest<5.4
+pytest-cov
+pytest-sugar
+check-manifest
+pydocstyle
+flake8
```

Comment:
why do we need a requirements.txt? Github actions doesn't work with environment.yml?

Reply:
installing with pip takes 1 mins on CI vs 5 mins using conda. It was just faster and conda is tested with circleci

Reply:
It’s because we download and install miniconda first I suppose. But there’s also the option to use caching